content/en/docs/develop/porting.md

```diff
@@ -152,8 +151,8 @@ Let's first start by initializing a working environment for ourselves:
    We can now use `kraft` to initialize a template library for us:
 
    ```bash
```

Comment:
Remove `bash`. We will only be using `bash` for shell scripts, not commands, since the rendering will include the output, not just the command. If the output uses Bash keywords, it will be highlighted although it's not useful.

Reply:
The linter will complain if there is no language specified within code snippets.
Since we will add the `$` prefix on all commands after #129 is merged, I would say we either add `bash` and let the command and output be separated by the `$` prefix or add a separate text snippet for the command output.

Reply:
Yes, there is the `text` "language" for the code snippets.

Reply:
We can also remove the code snippets language check, and we will need to ask the reviewers to check the existence of a language when necessary.